src/headless/tasks/inasafe_wrapper.py

```diff
@@ -4,6 +4,11 @@
 
 from headless.celery_app import app, start_inasafe
 from headless.tasks import inasafe_analysis
+from headless.settings import (
```

Comment:
I think you can delete this.

Reply:
Why? Are you suggesting to get it from environment variable in run time?